presto-main/src/test/java/io/prestosql/sql/planner/assertions/BasePlanTest.java

```diff
@@ -231,4 +253,20 @@ protected SubPlan subplan(String sql, LogicalPlanner.Stage stage, boolean forceS
     {
         LocalQueryRunner get();
     }
+
+    public class MetadataGetter
+    {
+        public Optional<TableHandle> getTableHandle(Session session, QualifiedObjectName objectName)
```

Comment:
maybe instead of `MetadataGetter ` add `protedcted getTableHandle` method to `BasePlanTest? WDYT?

Reply:
@kokosing Hmm, I thought that if in future we want to add more methods involving metadata, MetadataGetter will provide an umbrella abstraction for that, and BasePlanTest class will remain a bit cleaner. But it should be okay for now, I guess, since there're just a couple of methods.

Reply:
Then maybe to have `BasePlanTest` remain clean you could introduce `BasePushdownPlanTest`?